### DISCUSSION TITLE
CP2077 1.62 hotfix

### DISCUSSION THREAD

**Discussion by LeGrosWinnie** [wopss/RED4ext#24] [Q&A] [answered]
Hello,

Does it really work with this game version/patch ?
I switch from 1.11 to 1.13 as some mods where not loaded, and now the whole game is crashing.
Does it come from mods ? Red4ext ?

Thanks.

**Comment by wopss** [comment, marked as answer]
Yes, latest version of RED4ext is compatibile with 1.62 hotfix 1. You can check the log file at `game_directory/red4ext/logs/red4ext.log`.

**Reply by LeGrosWinnie** [reply]
Hello,
Yes I noticed that after futher investigation. I saw a mod was the crash issue.
As the mod was not working because of older version of Red3Ext the game was working but, even if mod author pretend it works fine..., in fact the mod is not compatible with 1.62 Hotfix.
Thanks again !
